app/v1/auth/keeper.go

```diff
@@ -271,6 +272,88 @@ func (am AccountKeeper) DecreaseTotalLoosenToken(ctx sdk.Context, coins sdk.Coin
 		"decreaseCoins", decreaseCoins.String(), "totalLoosenToken", totalLoosenToken.String())
 }
 
+// Turn a token demom to key used to get it from the account store
+func TotalSupplyStoreKey(denom string) []byte {
+	return append(totalSupplyKeyPrefix, []byte(denom)...)
```

Comment:
The same `token_id` that has been used to construct the key to store an asset might be a better choice -- reusable code `GetKeyIDFromUniqueID`, more consistency, and support for prefix search (although we might not need it for total supply, it's still good to have).

Reply:
Done